lib/clowne/ext/record_key.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Clowne
+  module Ext
+    module RecordKey # :nodoc: all
+      def key(record)
```

Comment:
A good candidate for refinements))

```ruby
refine Object do
  def to_key
    # ...
  end
end
```

Reply:
I wanna print `Refinements Maniac` T-shirt for you :)

I think `refine` for `Object` is too much for this case, no? 🤔

Reply:
>  is too much for this case

It couldn't be too much)